src/main/java/ee/carlrobert/codegpt/toolwindow/chat/ChatToolWindowTabPanelEditor.java

```diff
@@ -136,7 +175,47 @@ public void actionPerformed(@NotNull AnActionEvent event) {
     }));
     return wrapper;
   }
-
+  public void openDiffView(@NotNull Project project, Editor editor, String text) {
```

Comment:
Can we move this function to `DiffUtils.java`? Furthermore, it would be nice to split the logic into smaller units.